packages/admin/resources/views/components/layouts/app/sidebar/item.blade.php

```diff
@@ -36,7 +37,11 @@
             'bg-primary-500 text-white' => $active,
         ])
     >
-        <x-dynamic-component :component="$icon" class="h-5 w-5 shrink-0" />
+        @if($activeIcon && $active)
+            <x-dynamic-component :component="$activeIcon" class="h-5 w-5 shrink-0" />
+        @else
+            <x-dynamic-component :component="$icon" class="h-5 w-5 shrink-0" />
+        @endif
```

Comment:
Could you merge this using a conditional for `:component`?

Reply:
Already did

Reply:
How do you mean?
It could just be on a single line, since the passed icon component is the only thing that depends on the condition.

Reply:
`<x-dynamic-component :component="$active && $activeIcon ? $activeIcon : $icon" class="h-5 w-5 shrink-0" />`